cinema_service/settings.py

```diff
@@ -136,3 +136,9 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+REST_FRAMEWORK = {
+    "DEFAULT_PAGINATION_CLASS":
+        "rest_framework.pagination.LimitOffsetPagination",
+    "PAGE_SIZE": 10
```

Comment:
I recommend you to delete this and write your own custom paginator. It will be very easy, if you watched and read the lessons carefully ofc)
Moreover, try to use this new paginator ONLY with OrderViewSet. Because otherwise tests will fail :(